.github/workflows/build.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        node_version: ['10', '12']
+        node_version: ['10', '12', '14']
```

Comment:
Why adding this node14 support?

Reply:
`--experimental-loader` option is not supported before node 14.

Reply:
See https://nodejs.org/dist/latest-v12.x/docs/api/esm.html#esm_experimental_loaders, it seems working at node12, too.

Reply:
This option is enabled in node 12 and needs to be used in conjunction with `-experimental-modules', but `--experimental-modules` is deprecated in node 14.

Reply:
Maybe we can add backwards compatible unit tests.

Reply:
I think we can drop the node10, and use different flags for 12 and 14.

Reply:
fixed